pkg/plugins/commands.go

```diff
@@ -14,11 +15,15 @@ func Commands(l logger.Interface, s schema.Stage, fs vfs.FS, console Console) er
 	for _, cmd := range s.Commands {
 		out, err := console.Run(templateSysData(l, cmd))
 		if err != nil {
-			l.Error(out, ": ", err.Error())
 			errs = multierror.Append(errs, err)
 			continue
 		}
-		l.Info(fmt.Sprintf("Command output: %s", string(out)))
+		if strings.TrimSpace(out) != "" {
+			l.Info(fmt.Sprintf("Command output: %s", out))
```

Comment:
this made ALWAYS log a command output, even when the output was empty, thus polluting the logs. Still left on the debug log as its useful if we expect output

Reply:
Seems it's still on `Info`? 
```suggestion
			l.Debug(fmt.Sprintf("Command output: %s", out))
```